src/prometheus.rs

```diff
@@ -31,12 +31,17 @@ pub async fn serve_prometheus(registry: Registry, port: u16) {
 
 #[derive(Debug, Clone)]
 pub struct Stats {
-    /// The last exchange rate read from bitfinex.
-    exchange_rate_read:           Gauge,
-    /// The last exchange rate read from bitfinex.
-    exchange_rate_updated:        Gauge,
+    /// The last exchange rate read from each source.
+    /// Expects 1 label, the source's label.
+    exchange_rate_read:           GaugeVec,
+    /// The value of the last exchange rate update performed on chain.
+    exchange_rate_updated:        GaugeVec,
```

Comment:
Hmm, I guess it's illegal, I have added it because Felice was complaining about the value being 0 before we do the first update, and this vector doesn't allow it to be collected before then.

Reply:
I have changed it back to a Gauge, now with a wrapper, for the same effect, without going against the dependencies intended use.